pkg/cmd/init.go

```diff
@@ -95,19 +95,19 @@ func certSetup(cfg *config.MicroshiftConfig) (*certchains.CertificateChains, err
 				},
 				UserInfo: &user.DefaultInfo{Name: "system:kube-scheduler"},
 			},
-			&cryptomaterial.ClientCertificateSigningRequestInfo{
-				CertificateSigningRequestInfo: cryptomaterial.CertificateSigningRequestInfo{
+			&certchains.ClientCertificateSigningRequestInfo{
+				CSRMeta: certchains.CSRMeta{
 					Name:         "cluster-policy-controller",
-					ValidityDays: cryptomaterial.ClientCertValidityDays,
+					ValidityDays: cryptomaterial.ShortLivedCertificateValidityDays,
 				},
 				UserInfo: &user.DefaultInfo{Name: "system:kube-controller-manager"},
 			},
-			&cryptomaterial.ClientCertificateSigningRequestInfo{
-				CertificateSigningRequestInfo: cryptomaterial.CertificateSigningRequestInfo{
+			&certchains.ClientCertificateSigningRequestInfo{
+				CSRMeta: certchains.CSRMeta{
 					Name:         "route-controller-manager",
-					ValidityDays: cryptomaterial.ClientCertValidityDays,
+					ValidityDays: cryptomaterial.ShortLivedCertificateValidityDays,
 				},
-				UserInfo: &user.DefaultInfo{Name: "system:route-controller-manager"},
+				UserInfo: &user.DefaultInfo{Name: "system:serviceaccount:openshift-route-controller-manager:route-controller-manager-sa"},
```

Comment:
see `k8s.io/apiserver/pkg/authentication/serviceaccount` for name/groups setting up.

Note that you should add the `system:authenticated` group to the groups `MakeGroupNames()` function returns